tests/dummy/app/templates/docs/buttons.md

```diff
@@ -0,0 +1,40 @@
+# Buttons
+
+`ember-caluma` provides buttons for mutating the state of work items.
+
+## Usage
+
+To use these component call them with the required parameters.
```

Comment:
This sentence doesn't help at all. Delete it.

Reply:
Removed